description:
;;; files.el --- file input and output commands for Emacs
----------------------------
revision 1.938
date: 2007-11-02 11:03:42 +0000;  author: rfrancoise;  state: Exp;  lines: +2 -2;  commitid: wgxFxIhiWr3NtZDs;
2007-11-02  Drake Wilson  <drake@begriffli.ch>  (tiny change)

        * files.el (hack-local-variables): Fix membership tests to avoid
        treating all variables as safe if `enable-local-variables' is
        set to :safe.
=============================================================================
Index: lisp/files.el
===================================================================
RCS file: /sources/emacs/emacs/lisp/files.el,v
retrieving revision 1.937
retrieving revision 1.938
diff -u -r1.937 -r1.938
--- lisp/files.el	26 Oct 2007 14:25:17 -0000	1.937
+++ lisp/files.el	2 Nov 2007 11:03:42 -0000	1.938
@@ -2821,8 +2821,8 @@
 		;; If caller wants only the safe variables,
 		;; install only them.
 		(dolist (elt result)
-		  (unless (or (memq (car elt) unsafe-vars)
-			      (memq (car elt) risky-vars))
+		  (unless (or (member elt unsafe-vars)
+			      (member elt risky-vars))
 		    (hack-one-local-variable (car elt) (cdr elt))))
 	      ;; Query, except in the case where all are known safe
 	      ;; if the user wants no quuery in that case.
